dynatrace/configuration_v1/anomaly_detection_process_groups.py

```diff
@@ -0,0 +1,43 @@
+from typing import List, Optional, Dict, Any
+from requests import Response
+
+from dynatrace.environment_v2.schemas import ConfigurationMetadata
+from dynatrace.environment_v2.monitored_entities import EntityShortRepresentation
+from dynatrace.pagination import PaginatedList
+from dynatrace.dynatrace_object import DynatraceObject
+from dynatrace.http_client import HttpClient
+
+class AvailabilityMonitoringPG(DynatraceObject):
+    def _create_from_raw_data(self, raw_element: Dict[str, Any]):
+        self.method: str = raw_element.get("method")
```

Comment:
@jimm-with-a-j 

This is type should be the `Enum` I mentioned earlier as well instead of str, to show that only some values are possible.

Example: https://github.com/dlopes7/dt-api/blob/69bb84784da6cb7861b9d0eeeb226bcd8c0af143/dynatrace/configuration_v1/extensions.py#L241